lib/AST/GenerateAST.cpp

```diff
@@ -138,11 +139,11 @@ clang::Expr *GenerateAST::CreateEdgeCond(llvm::BasicBlock *from,
       auto br = llvm::cast<llvm::BranchInst>(term);
       if (br->isConditional()) {
         // Get the edge condition
-        result = ast_gen.GetOperandExpr(br->getCondition());
+        result = ast_gen.GetOperandExpr(*(br->op_end() - 3));
```

Comment:
Is there some other way to get the llvm::Use that would be more easily understandable? The indexes are kind of opaque to me personally.